Procfile

```diff
@@ -39,7 +39,7 @@ celery_scheduler: REMAP_SIGTERM=SIGQUIT newrelic-admin run-program celery beat -
 pulse_listener_pushes: newrelic-admin run-program ./manage.py pulse_listener_pushes
 pulse_listener_tasks: newrelic-admin run-program ./manage.py pulse_listener_tasks
```

Comment:
Add the new listener here.
I will make sure that when we deploy this in Heroku it will be set to 0 (so there's no double ingestion happening).